OrbitGl/GlCanvas.cpp

```diff
@@ -362,95 +318,75 @@ void GlCanvas::Render(int a_Width, int a_Height) {
   m_NeedsRedraw = false;
   ui_batcher_.StartNewFrame();
 
-  ScopeImguiContext state(m_ImGuiContext);
+  ScopeImguiContext state(im_gui_context_);
 
   Timer timer;
   timer.Start();
 
-  prepareGlState();
-  prepare2DViewport(0, 0, getWidth(), getHeight());
+  PrepareGlState();
+  Prepare2DViewport(0, 0, GetWidth(), GetHeight());
 
   glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
 
   glBindTexture(GL_TEXTURE_2D, 0);
   glUseProgram(0);
 
   // Clear text renderer
-  m_TextRenderer.Init();
-  m_TextRenderer.Clear();
+  text_renderer_.Init();
+  text_renderer_.Clear();
 
   Draw();
 
   // We have to draw everything collected in the batcher at this point,
-  // as prepareScreenSpaceViewport() changes the coordinate system.
+  // as PrepareScreenSpaceViewport() changes the coordinate system.
   ui_batcher_.Draw(GetPickingMode() != PickingMode::kNone);
   ui_batcher_.ResetElements();
 
-  prepareScreenSpaceViewport();
+  PrepareScreenSpaceViewport();
 
   DrawScreenSpace();
 
   // Draw remaining elements collected with the batcher.
   ui_batcher_.Draw(GetPickingMode() != PickingMode::kNone);
 
-  m_TextRenderer.Display(&ui_batcher_);
+  text_renderer_.Display(&ui_batcher_);
   RenderText();
   RenderUI();
 
   glFlush();
-  cleanupGlState();
+  CleanupGlState();
 
   timer.Stop();
 
-  m_ImguiActive = ImGui::IsAnyItemActive();
+  im_gui_active_ = ImGui::IsAnyItemActive();
 
   PostRender();
 
   picking_ = false;
-  m_DoubleClicking = false;
+  double_clicking_ = false;
 }
 
-void GlCanvas::Resize(int a_Width, int a_Height) {
-  m_Width = a_Width;
-  m_Height = a_Height;
+void GlCanvas::Resize(int width, int height) {
+  width_ = width;
+  height_ = height;
   NeedsRedraw();
 }
 
-Vec2 GlCanvas::ToScreenSpace(const Vec2& a_Point) {
-  float x = (a_Point[0] / m_WorldMinWidth) * m_Width;
-  float y = (a_Point[1] / m_WorldHeight) * m_Height;
-
-  return Vec2(x, y);
-}
-
-Vec2 GlCanvas::ToWorldSpace(const Vec2& a_Point) {
-  float x = (a_Point[0] / m_Width) * m_WorldMinWidth;
-  float y = (a_Point[1] / m_Height) * m_WorldHeight;
-
-  return Vec2(x, y);
-}
-
-void GlCanvas::AddText(const char* a_Text, float a_X, float a_Y, float a_Z, const Color& a_Color,
-                       float a_MaxSize, bool a_RightJustified) {
-  m_TextRenderer.AddText(a_Text, a_X, a_Y, a_Z, a_Color, a_MaxSize, a_RightJustified);
-}
-
-int GlCanvas::AddText2D(const char* a_Text, int a_X, int a_Y, float a_Z, const Color& a_Color,
-                        float a_MaxSize, bool a_RightJustified, bool a_InvertY) {
-  return m_TextRenderer.AddText2D(a_Text, a_X, a_Y, a_Z, a_Color, a_MaxSize, a_RightJustified,
-                                  a_InvertY);
+void GlCanvas::AddText(const char* text, float x, float y, float z, const Color& color,
+                       float max_size, bool right_justified) {
```

Comment:
You probably must erase this function, because some previous change that I've made.